public/pages/Dashboard/index.scss

```diff
@@ -32,11 +32,11 @@
   width: 400px;
 }
 
-.full-screen {
+.dashboard-full-screen {
   position: absolute;
   top: 0;
   left: 0;
   width: 100%;
-  height: 100%;
+  height: 100vh;
```

Comment:
`vh` doesn't fix it well, so I change to current solution, setting `<EuiPage>` height to 100%. `100vh` makes sure the screen viewed by user is fully covered, but if page is too long, scrolling down will still show uncovered page. Example is below:

![Screen Shot 2020-05-28 at 5 30 59 PM](https://user-images.githubusercontent.com/59710443/83210347-7784e180-a10f-11ea-9cf8-3d00cbe180f3.png)

Height 100% does its work, so I also use the same css `full-screen` as detector detail page uses. But height 100% works if `<EuiPage>` height is 100%, as `root cause` explained above. Detail page has no such problem because its height is already very long.